src/variations.json

```diff
@@ -6,11 +6,19 @@
         "urlclassifier.trackingAnnotationTable": "test-track-simple,base-track-digest256",
         "network.cookie.cookieBehavior": 0
       },
+      "nonDefault": [
```

Comment:
Instead of this, how about:
```
"expectValues": {
  "browser.contentblocking.category": "standard"
}
```
And then also add it to `setValues`:
```
"browser.contentblocking.category": "custom",
```